apps/docs/src/guide/wallets/wallet-transferring.md

```diff
@@ -30,6 +30,12 @@ This method also creates a `ScriptTransactionRequest` and populates it with the
 
 <<< @/../../docs-snippets/src/guide/cookbook/transferring-assets.test.ts#transferring-assets-3{ts:line-numbers}
 
+## Transferring Assets To Multiple Wallets
```

Comment:
Potentially redundant, is it worth having a note somewhere just enforcing that you shouldn't use `transfer` or `batchTransfer` to try and transfer to contracts, and link to `transferToContract`?

Reply:
This section was already present on this doc page. I've just moved it to be on top of the `## Transferring Assets To Contracts`

Maybe we can move the sections related to contract transferring to another page if that makes more sense. Should this be addressed in this PR?

Reply:
Ah apologies, no I'm happy with it being on this page.

Just might be worth putting the bit about not using `transfer` for contracts inside a `> Note:`?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3335/commits/845e347f11e91efc2cd8b8155be98abc49cf44cb